agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/OpenTelemetryConfigurer.java

```diff
@@ -208,7 +211,40 @@ private static SpanExporter createSpanExporter(
       Configuration configuration,
       boolean captureHttpServer4xxAsError) {
 
-    SpanExporter spanExporter = new Exporter(telemetryClient, captureHttpServer4xxAsError);
+    AzureMonitorTraceExporter azureMonitorTraceExporter =
+        new AzureMonitorTraceExporter(
+            captureHttpServer4xxAsError,
+            telemetryClient::populateDefaults,
```

Comment:
we can't pass TelemetryClient directly since that's in the javaagent code. will definitely revisit this large constructor parameter list once I change javaagent to use the live metrics span processor and get rid of the extra batching pipeline in the javaagent